pkg/models/models.go

```diff
@@ -714,3 +714,38 @@ var ExampleAccountRolesOrganizationReply = AccountRolesOrganizationReply{
 	Roles:          []string{"can_view", "can_edit", "audit_log_viewer"},
 	OrganizationID: "01J4HMNDSZCCQBTY93BF9CBF5D",
 }
+
+// =========
+// FILES
+// =========
+
+type UploadFilesRequest struct {
+	UploadFile []byte `form:"uploadFile"`
```

Comment:
should this be `byte` or `multipart.File`

Reply:
I think `multipart.FileHeader`, updating